tokio/CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# 1.19.0 (June 3, 2022)
+
+### Added
+
+- runtime: add `is_finished` method for `JoinHandle` and `AbortHandle` ([#4709])
+- runtime: make global queue and event polling intervals configurable ([#4671])
+- sync: add `Notified::enable` ([#4705])
+- sync: add `watch::Sender::send_if_modified` ([#4591])
+- sync: add resubscribe method to broadcast::Receiver ([#4607])
+
+### Changed
+
+- io: refactor out usage of Weak in the io handle ([#4656])
+- metrics: correctly update atomics in `IoDriverMetrics` ([#4725])
```

Comment:
This seems to be better in the `## Fixed` section.

Reply:
I'll put it under `## Unstable`